.chloggen/3149-add-must-gather.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Add a must gather application to help troubleshoot"
+
+# One or more tracking issues related to the change
+issues: [3149]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: "The new application was added to the operator container so it can be run without adding any new container. It can be run from the /must-gather path"
```

Comment:
The must gather utility was added to the operator container image.

Could we explain here how the utility should be run? E.g. can user run it from the host and use the default kube config?